daemons/controld/controld_messages.c

```diff
@@ -381,6 +381,18 @@ relay_message(xmlNode * msg, gboolean originated_locally)
         return TRUE;
     }
 
+    // Get the message type appropriate to the destination subsystem
+    if (is_corosync_cluster()) {
+        dest = text2msg_type(sys_to);
+        if ((dest < crm_msg_ais) || (dest > crm_msg_stonith_ng)) {
```

Comment:
For the future, it'd be nice to have a _min and _max value for this enum so we don't have to remember what the bounds are.

Reply:
I used to like min/max entries, but I realized it makes it impossible to add values at the start or end without breaking API (for public enums). One possibility would be to define internal min/max symbols for them, but then they're in separate headers and easier to forget.